doc/embed.md

```diff
@@ -0,0 +1,11 @@
+# Embedding Worldview
+
+To embed NASA Worldview in your page, navigate to the share toolbar, select the `embed` tab, and copy the code that wraps the current Worldview instance in an `<iframe>` inline element. Add this HTML code to your website to embed Worldview. You will need to wrap the `<iframe>` code in a wrapper element (e.g., `<div>`) with your desired `width`, `height`, and any additional style changes.
```

Comment:
Users / embedders may not know what the "share toolbar" is so would suggest rewording to something like, "...click the 'share' button in the upper right corner of the app, select the 'embed' tab..."